developer-docs-site/docs/nodes/validator-node/operator/node-requirements.md

```diff
@@ -68,15 +68,20 @@ You can configure the port settings on your node using the configuration YAML fi
 
 ### Port settings
 
-For the validator:
+:::tip Default port settings
+The section below describes the default port settings used by the validators, validator fullnodes and public fullnodes. If you override these ports in your node config, be sure to change the ports in the section below accordingly.
```

Comment:
something e.g. "Several ports should be closed to the general internet, but the inspection/metrics port (9101) and REST API port (80 or 8080) may be useful to your internal network. Exposing the REST API endpoint could be a deliberate choice, but some other kind of auth or rate limiting should be put in front of it."

Reply:
I've expanded the current text to cover the additional points and also added a new `caution` bubble below that talks about public exposure of the services 😄